sagenb/notebook/compress/JavaScriptCompressor.py

```diff
@@ -55,7 +55,8 @@
 # @Credits		Dean Edwards for his originally idea [dean.edwards.name] and his JavaScript packer
 # @License		GNU General Public License (GPL)
 
-import re, time, string, SourceMap, BaseConvert
+import re, time, string

```

Comment:
Although now that I look, it only uses the `string` module for `string.join` which is deprecated, so might as well fix those too.